src/lib.rs

```diff
@@ -1,10 +1,23 @@
+//! A compiled-in copy of the root certificates trusted by Mozilla as of {:?}",
 //!
-//! This library is automatically generated from the Mozilla
-//! IncludedCACertificateReportPEMCSV report via ccadb.org. Don't edit it.
+//! To use this library with rustls 0.22:
```

Comment:
I think it would be safe to remove the "0.22" here - to get this to build in CI we need to take a dev-dependency on Rustls which will be versioned and our example building over time.